### PR TITLE
Slides e2e tests

### PR DESCRIPTION
**Pull request by olegrise**
@andrecardoso please review

### CODE REVIEW

File: test/e2e/template-editor/cases/components/slides.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      slidesComponentPage = new SlidesComponentPage();
+
+      presentationsListPage.loadCurrentCompanyPresentationList();
+
+      presentationsListPage.createNewPresentationFromTemplate('"Slides Component Test"', 'slides-component-test');
+    });
+
+    describe('basic operations', function () {
+
+      it('should auto-save the Presentation after it has been created', function () {
```
Comment (andrecardoso) [inline test/e2e/template-editor/cases/components/slides.js]: Do we really need to test this for every component?

Comment (olegrise) [reply test/e2e/template-editor/cases/components/slides.js]: @andrecardoso good point. I'll remove.

**Comment by olegrise** [comment]
@andrecardoso I removed the unnecessary test. Please review.

**Comment by andrecardoso** [comment]
Looks good. Thanks

**Pull request merged**